component/service/src/main/java/org/exoplatform/social/rest/api/EntityBuilder.java

```diff
@@ -1854,6 +1854,7 @@ public static SiteEntity buildSiteEntity(PortalConfig site,
                           site.getName(),
                           !StringUtils.isBlank(displayName) ? displayName : site.getName(),
                           StringUtils.isNotBlank(translateSiteDescription) ? translateSiteDescription : site.getDescription(),
+                          new UserNodeRestEntity(rootNode),
```

Comment:
see this [review ](https://github.com/exoplatform/layout-management/pull/250#pullrequestreview-1652348450)